src/View/AnonymousComponent.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Performing\TwigComponents\View;
+
+class AnonymousComponent extends Component
+{
+    private array $attributes;
+
+    public function __construct($attributes = [])
+    {
+        $this->attributes = $attributes;
+    }
+
+    public function template(): string
```

Comment:
Wouldn't it better if this was part of the Component class because it's possible that you might want to use this default logic even if you define your own Component and only specify the Component properties or a custom method but not necessarily care to override the location of the component? 

Then this AnonymousComponent wouldn't need to exist and Component wouldn't need to be Abstract and could be used directly as an anonymous one.